browser/chrome/browser/migration/migration.dtd

```diff
@@ -43,4 +44,4 @@
 <!ENTITY done.title                     "Importowanie zakończone">
 <!ENTITY done.label                     "Poniższe elementy zostały zaimportowane:">
 
-<!ENTITY closeSourceBrowser.label "Przed przejściem dalej, oroszę upewnić się, że wybrana przeglądarka jest wyłączona.">
+<!ENTITY closeSourceBrowser.label "Przed przejściem dalej proszę upewnić się, że wybrana przeglądarka jest wyłączona.">
```

Comment:
No the case or problem now but please remember that person taking care of changed files should review the changes.

Additionally, "Typo and missing comma in closeSourceBrowser.label" would make better commit message for https://github.com/piotrdrag/mozilla-l10n-pl/commit/82e9e080b94dd6978f79683f3af2b55439fad098